pages/folder/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import { useFetch } from "usehooks-ts";
+import { FolderLayout } from "@/components/FolderLayout";
+import { Folder } from "@/types/folder.type";
+
+function FolderPage() {
+  //임시 상수
+  const linksUrl = "/api/links";
+  const folderId = 1;
+
+  const [selected, setSelected] = useState<string>("전체");
+  const [url, setUrl] = useState(linksUrl);
+
+  const { data: folderNameData } = useFetch(`/api/folder/${folderId}`);
+  const { data: linksData } = useFetch<Folder>(url);
+
+  const onClick = (name: string, linksId?: number) => {
+    let url;
+    setSelected(name);
+    if (linksId) {
+      url = `${linksUrl}/${linksId}`;
+    } else {
+      url = linksUrl;
+    }
+
+    setUrl(url);
+  };
```

Comment:
```suggestion
const onClick = (name: string, linksId?: number) => {
    setSelected(name);
    setUrl(linksId ? `${linksUrl}/${linksId}` : linksUrl);
  };
```
linksId가 있으면 url을 `${linksUrl}/${linksId}` 로 하고,
없으면 linksUrl 로 한 url값을 setUrl로 보내는걸 
조건부 연산자로 간결하게 표현할 수 있겠네요.